package.json

```diff
@@ -1,10 +1,11 @@
 {
   "dependencies": {
-    "@ckeditor/ckeditor5-build-classic": "^11.0.1",
-    "@rails/webpacker": "3.5",
+    "@ckeditor/ckeditor5-build-classic": "12.3.1",
+    "@rails/webpacker": "4.0.7",
+    "core-js": "3",
     "fstream": "1.0.12",
-    "ideogram": "1.8.0",
-    "igv": "2.1.0",
+    "ideogram": "1.11.0",
+    "igv": "2.2.15",
     "jquery": "3.4.0",
     "jquery-ui": "1.12.1",
     "js-yaml": "3.13.1",
```

Comment:
Same thing here - I'd like to remove these and see if by upgrading to webpack 4 we can get onto later versions that will remove the need to declare these directly.

Reply:
Resolved in 87d7ac1.